xml/System.Formats.Asn1/AsnWriter.xml

```diff
@@ -757,10 +897,23 @@
         <Parameter Name="tag" Type="System.Nullable&lt;System.Formats.Asn1.Asn1Tag&gt;" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <param name="tag">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="value">The bits to write</param>
+        <param name="tag">The tag to write, or <see langword="null" /> for the default tag (Universal 3).</param>
+        <summary>Write a bit array value as a NamedBitList with a specified tag.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The index of the bit array corresponds to the bit number in the encoded format, which is different than the value produced by <xref:System.Collections.BitArray.CopyTo(System.Array,System.Int32)> with a byte array.
+              For example, the bit array `{ `false`, `true`, `true` }` encodes as `0b0110_0000` with 5 unused bits.
```

Comment:
Looks like the tool didn't do the best possible job here.

What it wrote (`{ `false`, `true`, `true` }`) doesn't print nicely.

```suggestion
              For example, the bit array `{ false, true, true }` encodes as `0b0110_0000` with 5 unused bits.
```

`{ false, true, true }`  is more what's expected.  (And matches the xml content... is there a late filter that wraps boolean values in backticks?  It might need to learn if it's already in an escaped context?)

Reply:
Yes, the tool finds and wraps `true` and `false` with backticks. It's not perfect. This is a rare case where the larger string was already escaped, I think I can get it fixed.